package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "to-do-list-app",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack serve"
```

Comment:
- [ ] Your project was failing to start from my end and was throwing the error ```Missing script: "start"```. On further inspection, I realized that you do not have the "start" script to enable someone to run your project from their local machines. To fix this, you simply need to add ```    "start": "webpack serve --open", ``` after "dev" and it will work perfectly.

**This is the error it throws without it:**

![image](https://user-images.githubusercontent.com/86318284/183716256-5967f690-06f5-4a37-be0d-65e1fcce457b.png)

**This is the correct scripts:**

```
"scripts": {
    "build": "webpack",
    "start": "webpack serve --open",
    "dev": "webpack serve"
  },
```